config/prod/JKG-SciComp-S3.yaml

```diff
@@ -26,4 +26,3 @@ parameters:
   # (Optional) Allow accounts, groups, and users to access bucket (default is no access).
   GrantAccess:
     - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
-    - 'arn:aws:sts::563295687221:assumed-role/AWSReservedSSO_Developer_baa6fed639faf5e7/jake.gockley@sagebase.org'
```

Comment:
As per conversation with Khai and Anna on slack, this bucket can be deleted completely